module/core/clone_dyn/src/lib.rs

```diff
@@ -35,22 +35,22 @@ pub( crate ) mod private
   #[ cfg( all( feature = "use_std", not( feature = "use_alloc" ) ) ) ]
   use std::boxed::Box;
 
-  /// Clone boxed dyn.
-  ///
-  /// Not intended to be used directly.
-  #[ inline ]
-  pub fn _clone_boxed< T >( t : &T ) -> Box< T >
-  where
-    T : ?Sized,
-  {
-    unsafe
-    {
-      let mut ptr = t as *const T;
-      let data_ptr = &mut ptr as *mut *const T as *mut *mut ();
-      *data_ptr = Box::into_raw( Box::new( < &T >::clone( &t ) ) ) as *mut ();
-      Box::from_raw( ptr as *mut T )
-    }
-  }
+  // /// Clone boxed dyn.
```

Comment:
why?

Reply:
was problem with including this code to derive tools tests. I returned this code and removed `clone_dyn_no_std` feature from derive tools, so now it works with this code.